public/app/plugins/panel/canvas/InlineEdit.tsx

```diff
@@ -0,0 +1,197 @@
+      builder.addNestedOptions(getLayerEditor(instanceState));
+
+      const selection = state.selected;
+      if (selection?.length === 1) {
+        const element = selection[0];
+        if (!(element instanceof GroupState)) {
+          builder.addNestedOptions(
+            getElementEditor({
+              category: [`Selected element (${element.options.name})`],
+              element,
+              scene: state.scene,
+            })
+          );
+        }
+      }
+    };
+
+    return getOptionsPaneCategoryDescriptor({}, supplier);
+  }, [instanceState]);
+
+  const onDragStop = (event: any, dragElement: any) => {
+    setPlacement({ x: dragElement.x, y: dragElement.y });
+    saveToStore(dragElement.x, dragElement.y, measurements.width, measurements.height);
+  };
+
+  const onResizeStop = (event: SyntheticEvent<Element, Event>, data: ResizeCallbackData) => {
+    const { size } = data;
+    setMeasurements({ width: size.width, height: size.height });
+    saveToStore(placement.x, placement.y, size.width, size.height);
+  };
+
+  const saveToStore = (x: number, y: number, width: number, height: number) => {
+    store.setObject(inlineEditKey, { x: x, y: y, w: width, h: height });
+  };
+  return (
+    <Draggable handle="strong" onStop={onDragStop} position={{ x: placement.x, y: savedPlacement.y }}>
+      <Resizable height={measurements.height} width={measurements.width} onResize={onResizeStop}>
+        <div
+          className={cx('box', 'no-cursor', `${styles.inlineEditorContainer}`)}
+          style={{ height: `${measurements.height}px`, width: `${measurements.width}px` }}
+          ref={ref}
+        >
+          <strong className={cx('cursor', `${styles.inlineEditorHeader}`)}>
+            <div className={styles.placeholder} />
+            <div>Canvas Inline Editor</div>
+            <IconButton name="times" size="xl" className={styles.inlineEditorClose} onClick={onClose} />
+          </strong>
+          <div style={{ overflow: 'scroll' }} className={styles.inlineEditorContentWrapper}>
+            <div className={styles.inlineEditorContent}>
+              <div>
+                <div>{pane.items.map((v) => v.render())}</div>
+                <div>
+                  {pane.categories.map((c) => {
+                    return (
+                      <div key={c.props.id}>
+                        <h5>{c.props.title}</h5>
+                        <div>{c.items.map((s) => s.render())}</div>
+                      </div>
+                    );
+                  })}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </Resizable>
+    </Draggable>
+  );
+};
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  inlineEditorContainer: css`
+    display: flex;
+    flex-direction: column;
+    background: ${theme.colors.panelBg};
+    box-shadow: 5px 5px 20px -5px #000000;
+    z-index: 1000;
```

Comment:
z-index needs to be higher than selecto....  
![Uploading 2022-05-02_09-55-30 (1).gif…]()

Reply:
ok -- this is fine for now then.  We can figure out how to modify selecto to have a lower z-index in a different PR

Reply:
I updated it.